### DISCUSSION TITLE
How to remove worker label from master nodes after installation

### DISCUSSION THREAD

**Discussion by ibotty** [okd-project/okd#1305] [General]
When a cluster grows and at one point should separate master and worker nodes, how is it done.  I could, of course, remove the `.../worker` label, but that will have to be repeated for newly installed (exchanged) master nodes and does not feel right.  What's the recommended way?

When answered, what's the best place in the documentation to add that information?  I will certainly write something up.

**Comment by vrutkovs** [comment]
https://docs.okd.io/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-master-schedulable_nodes-nodes-working

**Comment by ibotty** [comment]
Thank you, I don't know why I missed it.
